doc/source/whatsnew/v0.20.0.txt

```diff
@@ -59,6 +59,7 @@ Other enhancements
 - ``Series`` provides a ``to_excel`` method to output Excel files (:issue:`8825`)
 - The ``usecols`` argument in ``pd.read_csv`` now accepts a callable function as a value  (:issue:`14154`)
 - ``pd.DataFrame.plot`` now prints a title above each subplot if ``suplots=True`` and ``title`` is a list of strings (:issue:`14753`)
+-  The ``pd.Series.interpolate`` now supports timedelta as index type with method=`time` (:issue:`6424`)
```

Comment:
small edit, but I can do on merge:

remove the leading ``The`` and ``method='time'`` needs to be in double backticks.

``timedelta as *an* index type`` (add an)